deploy/helm/fluent-bit-overrides.yaml

```diff
@@ -22,70 +22,75 @@ tolerations:
     operator: Exists
 
 input:
-  tail:
-    memBufLimit: 5MB
-    parser: docker
-    path: /var/log/containers/*.log
   systemd:
     enabled: true
-    filters:
-      systemdUnit:
-        - addon-config.service
-        - addon-run.service
-        - cfn-etcd-environment.service
-        - cfn-signal.service
-        - clean-ca-certificates.service
-        - containerd.service
-        - coreos-metadata.service
-        - coreos-setup-environment.service
-        - coreos-tmpfiles.service
-        - dbus.service
-        - docker.service
-        - efs.service
-        - etcd-member.service
-        - etcd.service
-        - etcd2.service
-        - etcd3.service
-        - etcdadm-check.service
-        - etcdadm-reconfigure.service
-        - etcdadm-save.service
-        - etcdadm-update-status.service
-        - flanneld.service
-        - format-etcd2-volume.service
-        - kube-node-taint-and-uncordon.service
-        - kubelet.service
-        - ldconfig.service
-        - locksmithd.service
-        - logrotate.service
-        - lvm2-monitor.service
-        - mdmon.service
-        - nfs-idmapd.service
-        - nfs-mountd.service
-        - nfs-server.service
-        - nfs-utils.service
-        - node-problem-detector.service
-        - ntp.service
-        - oem-cloudinit.service
-        - rkt-gc.service
-        - rkt-metadata.service
-        - rpc-idmapd.service
-        - rpc-mountd.service
-        - rpc-statd.service
-        - rpcbind.service
-        - set-aws-environment.service
-        - system-cloudinit.service
-        - systemd-timesyncd.service
-        - update-ca-certificates.service
-        - user-cloudinit.service
-        - var-lib-etcd2.service
-    maxEntries: 1000
-    readFromTail: true
-    tag: host.*
-
-filter:
-  kubeTag: containers
 
 rawConfig: |-
   @INCLUDE fluent-bit-service.conf
-  @INCLUDE fluent-bit-input.conf
-  @INCLUDE fluent-bit-output.conf
+
+  [INPUT]
```

Comment:
Will `extraEntries.input` help to expose the knobs we want?

Reply:
Unfortunately not. There's two issues with `extraEntries.input`:
1. Since `Refresh_Interval 5` is already hard coded in line 33, we cannot configure this parameter.
2. Since `extraEntries.input` is on line 50, after the systemd section, any configuration will be inserted to the systemd section, not the tail section. So if we want to specify `Rotate_Wait` with `extraEntries.input`, we cannot use the systemd section (`input.systemd.enabled` must be set to false).